Models/Location.ts

```diff
@@ -2,8 +2,8 @@ interface Location {
     City: string,
     Territory: string,
     Country: string,
-    XCoord: number,
-    YCoord: number
+    Longitude: number,
```

Comment:
We should probably refactor our stuff to have latitude in front of longitude in all cases given cardinal coordinates are always X, Y. -Except for that one external api call >:(

Reply:
Longitude is the X one brother